qa/suites/rgw/dbstore/cluster.yaml

```diff
@@ -0,0 +1,10 @@
+roles:
+- [mon.a, osd.0, osd.1, osd.2, mgr.0, client.0]
+openstack:
```

Comment:
I'm curious, what's the point of having the openstack task here?

Reply:
I had referred to  couple of other test-suites. Will fix this and below comments.